backend/src/api/models/surveys/survey.py

```diff
@@ -68,6 +69,21 @@ class Config:
         extra = "forbid"
 
 
+class SurveyRespondentsFetchInput(BaseModel):
```

Comment:
this is exactly the same as SurveyStructureFetchInput - let's generalize it and use the same one in all places